src/icesat2_tracks/analysis_db/A01b_ALT07_SHNH_variance_tester.py

```diff
@@ -1,6 +1,6 @@
 import os, sys
 #execfile(os.environ['PYTHONSTARTUP'])
-
+from icesat2_tracks.config import IceSAT2_startup as config
```

Comment:
Maybe import `mconfig` directly? – Then you don't need to change any of the code which uses it – it would be a drop-in replacement.
```suggestion
from icesat2_tracks.config.IceSAT2_startup import mconfig
```